pekko-connectors-sample-jdbc-to-elasticsearch/step_001_complete/src/main/java/samples/javadsl/Main.java

```diff
@@ -1,31 +1,39 @@
 /*
- * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
Revert all license changes. We need to keep the Lightbend headers.

Reply:
done

Reply:
I have removed apache license headers keep it. Friendly question: Why not add the Apache license header?

Reply:
When a source file already has a license header, you normally just keep it. It's a big legal mine field and it's been one of the most complicated aspects of getting Pekko up and running. I am busy this weekend and don't have time to study the licenses. I will look at them next week.